src/Microsoft.ML.Data/Prediction/CalibratorCatalog.cs

```diff
@@ -425,24 +419,24 @@ public sealed class PavCalibratorEstimator : CalibratorEstimatorBase<PavCalibrat
 
         [BestFriend]
         private protected override CalibratorTransformer<PavCalibrator> Create(IHostEnvironment env, PavCalibrator calibrator)
-            => new PavCalibratorTransformer(env, calibrator);
+            => new PairAdjacentViolatorsCalibratorTransformer(env, calibrator);
 
     }
 
     /// <summary>
-    /// The <see cref="ITransformer"/> implementation obtained by training a <see cref="PavCalibratorEstimator"/>
+    /// The <see cref="ITransformer"/> implementation obtained by training a <see cref="PoolAdjacentViolatorsCalibratorEstimator"/>
```

Comment:
>PoolAdjacentViolatorsCalibratorEstimator [](start = 88, length = 40)

What is the difference between `PoolAdjacentViolators` and `PairAdjacentViolators`? They both seem to be an expansion of the single acronym that used to just be PAV, so I'm wondering if this distinction is on purpose.

Reply:
Changed to Isotonic to avert confusion

---
In reply to: [262238146](https://github.com/dotnet/machinelearning/pull/2766#discussion_r262238146) [](ancestors = 262238146)